api/src/test/java/org/confeti/controllers/ControllersTests.java

```diff
@@ -0,0 +1,10 @@
+package org.confeti.controllers;
+
+public final class ControllersTests {
+    public static final String STATUS = "{\"status\":\"OK\"}";
+
+
+    private ControllersTests() {
+        throw new UnsupportedOperationException();
```

Comment:
I prefer to just leave a comment `not instantiable`. Joshua Bloch from Effective Java [offers](https://stackoverflow.com/questions/25658330/why-java-util-objects-private-constructor-throws-assertionerror) to throw `AssertionError` and include a comment.